### PR TITLE
refactor(ava/insight): format code and fix types

### PR DESCRIPTION
**Pull request by Esorakouki**
### PR includes

format code and fix types

### CODE REVIEW

File: packages/ava/package.json [hunk 1]
```diff
@@ -54,10 +54,19 @@
   },
   "dependencies": {
     "@antv/antv-spec": "^0.1.0-alpha.18",
+    "@stdlib/stats": "^0.0.12",
```
Comment (pddpd) [inline packages/ava/package.json:57]: we will replace it by data/stats

File: packages/ava/src/insight/algorithms/base/compare.ts [hunk 2]
```diff
@@ -1,4 +1,10 @@
-export const ascending = (left: any, right: any) => {
+/**
+ * lodash sortby asc function
+ * @param left any
+ * @param right any
+ * @returns number
+ */
+export function ascending(left: any, right: any) {
```
Comment (pddpd) [inline packages/ava/src/insight/algorithms/base/compare.ts:7]: Should we modify all the `any` to `unknown`?

File: packages/ava/src/insight/algorithms/change-point/bayesian.ts [hunk 3]
```diff
@@ -2,6 +2,8 @@ import BayesianChangePoint, { BreakPoint } from 'bayesian-changepoint';
 
 import { calcPValue } from './pettitt-test';
 
+import type { ChangePointItem } from './types';
+
 const breakpointVerifier = (next: BreakPoint<number>, prev: BreakPoint<number>): boolean => {
```
Comment (pddpd) [inline packages/ava/src/insight/algorithms/change-point/bayesian.ts:7]: why is it arrow function but L18 is function

File: packages/ava/src/insight/insights/extractors/lowVariance.ts [hunk 4]
```diff
@@ -14,20 +14,20 @@ type LowVarianceParams = {
 // Coefficient of variation threshold
 const CV_THRESHOLD = 0.15;
 
-export const findLowVariance = (values: number[], params?: LowVarianceParams): LowVarianceItem => {
-  const cv = statistics.coefficientOfVariance(values);
+export const findLowVariance = (values: number[], params?: LowVarianceParams): LowVarianceItem | null => {
```
Comment (pddpd) [inline packages/ava/src/insight/insights/extractors/lowVariance.ts:17]: arrow function?

File: packages/ava/src/insight/insights/extractors/majority.ts [hunk 5]
```diff
@@ -13,7 +13,7 @@ type MajorityParams = {
 
 const DEFAULT_PROPORTION_LIMIT = 0.6;
 
-export const findMajority = (values: number[], params?: MajorityParams): MajorityItem => {
+export const findMajority = (values: number[], params?: MajorityParams): MajorityItem | null => {
```
Comment (pddpd) [inline packages/ava/src/insight/insights/extractors/majority.ts:16]: Do u have to use the arrow function? There are some cases like this in these files

File: packages/ava/src/insight/insights/util.ts [hunk 6]
```diff
@@ -1,10 +1,11 @@
...
+import { mean } from 'lodash';
 import { cdf } from '@stdlib/stats/base/dists/normal';
 
+import { standardDeviation } from '../../data/statistics';
+
 export const calculatePValue = (values: number[], target: number) => {
-  const mean = _mean(values);
-  const std = statistics.standardDeviation(values);
-  const pValue = 1 - cdf(target, mean, std);
+  const meanValue = mean(values);
```
Comment (pddpd) [inline packages/ava/src/insight/insights/util.ts:7]: there is a mean function in data/stats

File: packages/ava/src/insight/types.ts [hunk 7]
```diff
@@ -1,4 +1,4 @@
-import { IPhrase } from '@antv/text-schema';
+import { NtvTypes } from '../ntv';
```
Comment (pddpd) [inline packages/ava/src/insight/types.ts:1]: import type or import, plz take a look
[feat(ava-react/ntv): finish ntv comp migrate. ](https://github.com/antvis/AVA/pull/496#discussion_r1024224093)

File: packages/ava/src/insight/algorithms/base/compare.ts (hunk 2 above)
Comment (Esorakouki) [reply packages/ava/src/insight/algorithms/base/compare.ts]: done

File: packages/ava/src/insight/insights/extractors/majority.ts (hunk 5 above)
Comment (Esorakouki) [reply packages/ava/src/insight/insights/extractors/majority.ts]: done

File: packages/ava/src/insight/insights/util.ts (hunk 6 above)
Comment (pddpd) [reply packages/ava/src/insight/insights/util.ts:7]: TODO: improve data/stats mean to process `undefined`

File: packages/ava/src/insight/narrative/utils/phrases-builder.ts [hunk 8]
```diff
@@ -1,5 +1,4 @@
-import { IPhrase, IEntityType } from '@antv/text-schema';
-
+import { NtvTypes } from '../../../ntv';
```
Comment (pddpd) [inline packages/ava/src/insight/narrative/utils/phrases-builder.ts:1]: here is one more

Comment (Esorakouki) [reply packages/ava/src/insight/narrative/utils/phrases-builder.ts]: done

**Review by pddpd — APPROVED** [review]
LGTM

**Pull request merged**